app/jinja_filters.py

```diff
@@ -193,6 +193,32 @@ def should_wrap_with_fieldset_processor():
     return {"should_wrap_with_fieldset": should_wrap_with_fieldset}
 
 
+@blueprint.app_template_filter()
+def get_width_class_for_number(answer):
+    allowable_widths = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20]
+
+    min_value = answer.get("min_value", {}).get("value", 0)
+    max_value = answer.get("max_value", {}).get("value")
+    if not max_value:
+        return None
+
```

Comment:
Petty comment, could/should be initialised when needed.
```suggestion
    max_value = answer.get("max_value", {}).get("value")
    if not max_value:
        return None

    allowable_widths = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20]
    min_value = answer.get("min_value", {}).get("value", 0)
```

Reply:
This isn't relevant now that there is no early return.